docker/docs/index.html

```diff
@@ -1,45 +1,112 @@
+    <p>
+      Climate change science has witnessed an explosion in the amount and types
+      of data that can be brought to bear on the potential responses of the
+      terrestrial carbon cycle and biodiversity to global change. Many of the
+      most pressing questions about global change are not necessarily limited by
+      the need to collect new data as much as by our ability to synthesize
+      existing data. This project specifically seeks to improve this ability.
+      Because no one measurement provides a complete picture, multiple data
+      sources must be integrated in a sensible manner. Process-based models
+      represent an ideal framework for integrating these data streams because
+      they represent multiple processes at different spatial and temporal scales
+      in ways that capture our current understanding of the causal connections
+      across scales and among data types. Three components are required to
+      bridge this gap between the available data and the required level of
+      understanding: 1) a state-of-the-art ecosystem model, 2) a workflow
+      management system to handle the numerous streams of data, and 3) a data
+      assimilation statistical framework in order to synthesize the data with
+      the model.
+    </p>
 
-<h2>Applications</h2>
+    <h2>Applications</h2>
 
-<ul>
-<li><a href="/pecan/">PEcAn web interface, from here you can run different models at different sites.</a></li>
-<li><a href="/bety/">BETY Interface to the database with all data used by the PEcAn system.</a></li>
-<li><a href="/rstudio/">RStudio Server, you can edit PEcAn files or run analysis on existing data.</a></li>
-<!--<li><a href="/shiny/">Visualization of outputs generated by PEcAn using shiny.</a></li>-->
-<li><a href="/pecan/history.php">PEcAn runs done previously.</a></li>
-<li><a href="/minio/">Browse the PEcAn data using minio graphicsal interface</a></li>
-<li><a href="/thredds/">Browse the PEcAn generated data using Thredds server</a></li>
-<!--<li><a href="/rstudio/">RStudio editor with PEcAn installed.</a></li>-->
-<!--<li><a href="/pecan/setups/">Basic Setups</a></li>-->
-</ul>
+    <ul>
+      <li>
+        <a href="/pecan/"
+          >PEcAn web interface, from here you can run different models at
```

Comment:
I'd prefer the `>` not to be on the next line (few other places as well)

Reply:
@robkooper @infotroph I went through there documentation. To fix this issue, we possibly have only two options:

-  We can add a comment <!-- prettier-ignore --> which will disable auto-formatting, but it's scope is up to the next HTML tag only. So we need to add this comment again & again.

-  We can increase the [print-width](https://prettier.io/docs/en/options.html#print-width) as this issue is happening due to line width constraints, but again it's not the right solution.

Please let me know if you have any other options?

Reply:
I'm ok with this break if it's driven by line length and the fix is a hack

Reply:
As per the discussion on slack. I am resolving this conversation